tcms/core/management/commands/initial_setup.py

```diff
@@ -20,4 +24,21 @@ def handle(self, *args, **kwargs):
         self.stdout.write("\n4. Setting permissions:")
         call_command("refresh_permissions", "--verbosity=%i" % kwargs["verbosity"])
 
+        self.stdout.write(" %i " % admins[0].id)
+
+        if 'tcms_tenants' in settings.INSTALLED_APPS:
+            self.stdout.write("\n5. Creating the public tenant:")
+            superusers = User.objects.filter(is_superuser=True).order_by('id')
```

Comment:
`.first()` will give you the first record from the query. 

No need to order b/c the natural order here is by ID and frankly we don't care that much about the order anyway. When this command is executed there should be only 1 super-user anyway.